volcli/volcli.go

```diff
@@ -453,9 +453,22 @@ func volumeRemove(ctx *cli.Context) (bool, error) {
 		return true, err
 	}
 
+	retry := ""
+	force := ""
+	if ctx.Bool("r") {
+		retry = "true"
+	}
+	if ctx.Bool("f") {
+		force = "true"
+	}
+
 	request := config.Request{
 		Policy: policy,
 		Volume: volume,
+		Options: map[string]string{
+			"retry": retry,
```

Comment:
you can do this with `fmt.Sprintf("%v", ctx.Bool("r"))` and I'd prefer that to the string manipulation to keep the code simpler.